src/api/admin.ts

```diff
@@ -32,6 +32,17 @@ export type ListCellIdsResponse = Array<CellId>
 export type ListActiveAppsRequest = void
 export type ListActiveAppsResponse = Array<InstalledAppId>
 
+export type AgentInfoSigned = {
+    agent: AgentPubKey,
+    signature: any, // don't yet have a type for signatures
+    agent_info: any, // don't yet have a type for agent_info
+}
+
+export type RequestAgentInfoRequest = { cell_id: CellId|null }
+export type RequestAgentInfoResponse = Array<AgentInfoSigned>
```

Comment:
```suggestion
export type RequestAgentInfoResponse = Buffer
```
I think this is what you want but not 100% sure. Whatever the equivalent of `Vec<u8>` is.

Reply:
see above